common/services/person.js

```diff
@@ -56,6 +58,25 @@ const personService = {
       .then(response => response.data)
   },
 
+  _getById(id, options = {}) {
+    if (!id) {
+      return Promise.reject(new Error(noPersonIdMessage))
+    }
+
+    return apiClient.find('person', id, options).then(response => response.data)
+  },
+
+  getById(id) {
+    const include = personService.defaultInclude
+    return personService._getById(id, { include })
+  },
+
+  async getCategory(id) {
+    const include = ['category']
+    const person = await personService._getById(id, { include })
+    return person.category
```

Comment:
This could also return a promise inline with the other service methods:

```javascript
getCategory(id) {
  return personService
    ._getById(id, { include: ['category'] })
    .then(person => person.category)
}
```